main.js

```diff
@@ -19,7 +19,26 @@ export async function decodeList({encodedList}) {
   return StatusList.decode({encodedList});
 }
 
-export async function createCredential({id, list}) {
+/**
+ * Creates a StatusList Credential.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.id - The id for StatusList Credential.
+ * @param {StatusList} options.list - An instance of StatusList.
+ * @param {string} options.statusPurpose - The purpose of the status entry.
```

Comment:
Should the `statusPurpose` here be an array of strings instead? Since a status list credential can have >= 1 credential status of different types "suspension" and "revocation".

Reply:
A single status list credential can only have one purpose. A credential can have multiple `credentialStatus` entries, each with a different `statusPurpose` pointing to a different status list credential (for that associated `statusPurpose`).